### PR TITLE
beef-kidney

### PR DESCRIPTION
**Pull request by Philip-Wittamore**

### CODE REVIEW

File: src/beef-kidney [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+- 🍳 Cook time: 45 min
+- 🍽️ Servings: 2
+
+## Ingredients
+
+- 1 beef kidney
+- 60gr butter 
+- 2 onions
+- 2 shallots
+- fresh parsley
```
Comment (DGKSK8LIFE) [inline src/beef-kidney:15]: can you quantify how much parsley?

File: src/beef-kidney [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+- 3 bay leaves
+- 400gr croutons or toasted bread in pieces
+
+## Directions
+
+1. clean the kidney and remove the fat
+2. cut the kidney into pieces
```
Comment (DGKSK8LIFE) [inline src/beef-kidney:22]: elaborate on how to cut the kidney

File: src/beef-kidney [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+3. put butter in a casserole dish
+4. add kidney, parsley, bay leaf, salt, pepper
+5. fry for 10 minutes while stirring
```
Comment (DGKSK8LIFE) [inline src/beef-kidney:25]: How to fry it?

**Review by DGKSK8LIFE — CHANGES_REQUESTED** [review]

File: src/beef-kidney [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+My wife's beef kidney recipe
+
+- ⏲️ Prep time: 10 min
+- 🍳 Cook time: 45 min
+- 🍽️ Servings: 2
+
+## Ingredients
+
+- 1 beef kidney
+- 60gr butter
```
Comment (hallzy) [inline src/beef-kidney:12]:
```suggestion
- 60g butter 
```

File: src/beef-kidney [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+
+## Ingredients
+
+- 1 beef kidney
+- 60gr butter 
+- 2 onions
+- 2 shallots
+- fresh parsley
+- 3 bay leaves
+- 400gr croutons or toasted bread in pieces
```
Comment (hallzy) [inline src/beef-kidney:17]:
```suggestion
- 400g croutons or toasted bread in pieces
```

File: src/beef-kidney [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+
+1. clean the kidney and remove the fat
+2. cut the kidney into pieces
+3. put butter in a casserole dish
+4. add kidney, parsley, bay leaf, salt, pepper
+5. fry for 10 minutes while stirring
+6. add 2 glasses of dry white wine
+7. add half a glass of water so that the mixture is well-bathed
+8. simmer for 35 minutes
+9. add about 400g of toasted bread in pieces
```
Comment (hallzy) [inline src/beef-kidney:29]:
```suggestion
9. Add in the croutons
```

**Review by hallzy — CHANGES_REQUESTED** [review]
In addition to the other comments, please rename your files:

`data/pix/beef-kidney.webP` -> `data/pix/beef-kidney.webp`

`src/beef-kidney` -> `src/beef-kidney.md`

File: src/beef-kidney (hunk 1 above)
Comment (Philip-Wittamore) [reply src/beef-kidney:15]: a sprig of fresh parsley

**Review by DGKSK8LIFE — APPROVED** [review]

**Pull request merged**